filetracker/client/client.py

```diff
@@ -285,9 +294,10 @@ def put_file(self,
             if (to_remote_store or not self.local_store) and self.remote_store:
                 versioned_name = self.remote_store.add_file(
                         name, filename, compress_hint=compress_hint)
+            delete_lock = False
```

Comment:
To może być za późno.
Co jeśli dodanie do local store się udało, a do remote nie?
Mamy plik w cache, a nie ma do niego locka.